frontend/public/models/index.ts

```diff
@@ -732,6 +732,7 @@ export const ClusterServiceBrokerModel: K8sKind = {
   namespaced: false,
   kind: 'ClusterServiceBroker',
   id: 'clusterservicebroker',
+  crd: true,
```

Comment:
Needed to make the keys in Redux models be `GroupVersionKinds`.

Reply:
Do the service catalog nav items and details page still work correctly with this change?

Reply:
Yes, I modified them to use `referenceForModel`.